Tests/PennyTests/GHHooksTests.swift

```diff
@@ -287,7 +287,7 @@ class GHHooksTests: XCTestCase {
                 hardLimit: 2_048,
                 trailingTextMinLength: 64
             )
-            XCTAssertMultilineStringsEqual(formatted, scalars_206)
+            XCTAssertMultilineStringsEqual(formatted, scalars_206 + "\n\(dots)")
```

Comment:
Why is `dots` factored out into a symbolic name here in the tests, but not in the source?

Reply:
Nothing too special, i don't think i saw the need really. there are 2 different places using it in the source code so i'm not sure if it's enough to make us facto it out.
The tests use it quite a lot there.